tox.ini

```diff
@@ -1,17 +1,9 @@
 [tox]
-envlist = py27, py34, coverage
+envlist = py27, py36
```

Comment:
This list gets used when using just ´tox` locally, so I have not listed all envs there.